packages/react-adminer/src/components/CellEditInput.tsx

```diff
@@ -53,26 +53,41 @@ const CellEditInput: React.FC<Props> = ({
 	const fieldConfig = getFieldByName(config, propertyName);
 	const isSaveDisabled = value === initValue || (fieldConfig?.nullable === false && value.length === 0);
 
+	const ref = useRef<HTMLInputElement>(null);
+
+	useEffect(() => {
+		document.addEventListener('click', handleClickOutside, true);
+	});
+
+	const handleClickOutside = (e: any): void => {
+		if (!ref.current!.contains(e.target)) {
+			setActiveRecord(undefined);
+		}
+		document.removeEventListener('click', handleClickOutside);
+	};
+
 	return (
-		<Space>
-			<Input
-				type="text"
-				defaultValue={initValue}
-				onKeyDown={event => {
-					if (event.key === 'Enter' && !isSaveDisabled) {
-						onSave();
-						event.preventDefault();
-						event.stopPropagation();
-					}
-				}}
-				onChange={v => {
-					setValue(v.target.value);
-				}}
-			/>
-			<Button type="primary" onClick={onSave} loading={isSaving} disabled={isSaveDisabled}>
-				{isSaving ? 'Saving...' : 'Save'}
-			</Button>
-		</Space>
+		<div ref={ref}>
```

Comment:
To je super, přesně takhle to mělo být.

Ale:
Přidal jsi sem nějakou logiku, co budeš dělat když to budeš chít na 20místech v apce? Budeš to pokaždé psát? Tohle je potřeba mít jako komponentu. 

Konkrétně se tomu říká HOC zkus si to vygooglit.

Reply:
a nebo pres hook